package.json

```
@@ -1,115 +1,86 @@
-    "npm": ">=6",
-    "node": ">=8"
-  },
-  "main": "convertBytesToHuman.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/z3hdro/2019-2-Atom-Frontend-E-Jumagaliyev.git"
-  },
-  "keywords": [],
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/z3hdro/2019-2-Atom-Frontend-E-Jumagaliyev/issues"
-  }
+	"name": "2019-2-Atom-Frontend-E-Jumagaliyev",
+	"homepage": "https://z3hdro.github.io/2019-2-Atom-Frontend-E-Jumagaliyev/",
+	"description": "Mail.Ru Group Frontend Atom course, 2019",
+	"version": "0.1.0",
+	"author": "E Jumagaliyev",
+	"dependencies": {
+		"@emotion/core": "^10.0.22",
+		"@emotion/styled": "^10.0.22",
+		"cross-env": "^6.0.3",
+		"node-sass": "^4.13.0",
+		"prop-types": "^15.7.2",
+		"react": "^16.11.0",
+		"react-dom": "^16.11.0",
+		"react-redux": "^7.1.1",
+		"react-router-dom": "^5.1.2",
+		"react-scripts": "^3.2.0",
+		"redux": "^4.0.4",
+		"redux-devtools-extension": "^2.13.8",
+		"redux-thunk": "^2.3.0"
+	},
+	"devDependencies": {
+		"@wemake-services/stylelint-config-scss": "^0.1.4",
+		"check-engine": "^1.8.0",
+		"eslint": "^6.6.0",
+		"eslint-config-airbnb": "^18.0.1",
+		"eslint-config-prettier": "^6.5.0",
+		"eslint-plugin-babel": "^5.3.0",
+		"eslint-plugin-import": "^2.18.2",
+		"eslint-plugin-jsx-a11y": "^6.2.3",
+		"eslint-plugin-prettier": "^3.1.1",
+		"eslint-plugin-react": "^7.16.0",
+		"husky": "^3.0.9",
+		"gh-pages": "^2.1.1",
+		"jscpd": "^2.0.15",
+		"prettier": "^1.18.2",
+		"pretty-quick": "^2.0.0",
+		"remark-cli": "^7.0.0",
+		"remark-lint": "^6.0.5",
+		"remark-preset-lint-markdown-style-guide": "^2.1.3",
+		"rosie": "^2.0.1",
+		"stylelint": "^11.1.1",
+		"stylelint-a11y": "^1.2.2",
+		"stylelint-config-css-modules": "^1.5.0",
+		"stylelint-no-unsupported-browser-features": "^3.0.2",
+		"stylelint-webpack-plugin": "^1.0.3"
+	},
+	"browserslist": [
+		"> 5%",
+		"last 2 versions",
+		"not ie < 11",
+		"not ie_mob < 11",
+		"not dead"
+	],
+	"scripts": {
+		"predeploy": "npm run build",
+		"deploy": "gh-pages -d build",
+		"start": "react-scripts start",
+		"build": "react-scripts build",
+		"eject": "react-scripts eject",
+		"format": "prettier --write '**/*.{js,jsx,json,md}'",
+		"format:changed": "pretty-quick",
+		"format:staged": "pretty-quick --staged",
+		"eslint-check": "eslint --print-config .eslintrc.js | eslint-config-prettier-check",
+		"lint": "npm run lint:js && npm run lint:dup",
+		"lint-fix": "eslint --fix",
+		"clean": "rm -rf dist",
+		"test": "npm run test:engine && npm run lint",
+		"test:engine": "check-engine",
+		"test:sec": "npm audit || true",
+		"test:unit": "cross-env NODE_ENV=test jest",
+		"lint:js": "eslint src/**/*.js -c .eslintrc.json",
+		"lint:css": "stylelint 'client/**/*.{scss,css}' && stylelint --rd 'client/**/*.{scss,css}'",
+		"lint:dup": "jscpd -g --threshold=1 client tests docker",
+		"ci": "npm run test && npm run build && npm run test:sec"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "pretty-quick --staged"
```

```suggestion
			"pre-commit": "npm run lint && npm run format:staged"
			
		
```

а вот это, чтобы линтер ругался перед коммитом. Когда будет ругань, надо будет править. Приучайся к использованию инструментов, чтобы писать более правильный код